### PR TITLE
Swapped dev and test environments

### PR DESCRIPTION
**Pull request by murdo-moj**
* Test is for testing on users
* Dev means development for developers
* So we should deploy new changes to dev first then test

### CODE REVIEW

File: .github/workflows/test.yml
Comment (tom-webber) [inline .github/workflows/test.yml:1]: rename file and workflow to 'tests and deploy'

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -82,19 +82,19 @@ jobs:
       - name: Run javascript tests
         run: npm test
 
-  # Only the deploy workflow should deploy to the test env on main
+  # Only the deploy workflow should deploy to the dev env on main
```
Comment (tom-webber) [inline .github/workflows/test.yml:85]:
```suggestion
  # Deploys to the 'dev' frontend unless pushed to main
```

**Review by tom-webber — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml
Comment (murdo-moj) [reply .github/workflows/test.yml:1]: the idea was that doing a deploy to dev is sometimes part of the test workflow you do - can I just leave the file name as is?

Comment (tom-webber) [reply .github/workflows/test.yml:1]: test is a confusing file (and workflow) name for something that deploys to dev

**Review by tom-webber — APPROVED** [review]

**Pull request merged**